hat/templates/iaso/pages/superset.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html>
+    <head>
+        <title>{{title}}</title>
+        {{ analytics_script | safe }}
+        <script src="https://unpkg.com/@superset-ui/embedded-sdk"></script>
```

Comment:
Ideally, we should have this copied in our code base and not depending on an external source